app/main.js

```diff
@@ -320,13 +319,14 @@ function getTrayMenu () {
     type: 'separator'
   })
 
-  if (!isPaused) {
+  if (breakPlanner == null || !breakPlanner.isPaused) {
```

Comment:
Yeah, `app.on('ready'` is what happens when app starts. Maybe also create function for tool tip updater? Might be more code, but easier to understand what happens when app starts. What do you think? 

I think we can go without checking for null.